flathub.json

```diff
@@ -0,0 +1,7 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
this is sad given it's an open source project. Can't you build from source?

Reply:
Official ARM64 Linux builds of Syphon exist, but they're currently a bit irregular. I will add ARM64 eventually though, and probably quite soon.